python/cuml/common/memory_utils.py

```diff
@@ -310,6 +310,11 @@ def _check_array_contiguity(ary):
         else:
             raise TypeError("No array_interface attribute detected in input. ")
 
+        # __cuda_array_interface__ v2 requires the strides to be omitted
+        # (either not set or set to None) for C-contiguous arrays.
```

Comment:
I would just change the wording of the comment, since the spec doesn't require strides to be omitted for C contiguous arrays, but _if_ the strides are not given or none, then the array is C-contiguous. So, a C-contiguous array can also have its strides set (correctly, implying its contiguity of course)

Reply:
True, I fixed it